zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/tasks/incident/IncidentUpdateRepositoryIT.java

```diff
@@ -775,15 +776,15 @@ void shouldGetFlowNodes() throws PersistenceException {
       batchRequest.executeWithRefresh();
 
       // when
-      final var flowNodes = repository.getFlowNodesInListView(List.of("1", "2"));
+      final var flowNodes = repository.getFlowNodeInstances(List.of("1", "2"));
```

Comment:
:clown_face: Looks like we made a mistake in a previous PR, and we were not testing the right thing - or rather, the production code was picking it up from the wrong index :smile: